backend/tests/e2e/conftest.py

```
@@ -0,0 +1,471 @@
+                "team": "platform"
+            },
+            "annotations": {
+                "finalizers": "kubernetes.io/pv-protection"
+            },
+            "timestamp": "2024-01-15T10:30:00Z"
+        }
+    }
+
+
+# Module-level cleanup for any global state that might leak
+def pytest_runtest_teardown(item, nextitem):
```

Ruff (0.12.2)

Unused function argument: `nextitem`

(ARG001)